packages/quantification/src/__tests__/quantification.test.ts

```diff
@@ -200,11 +203,86 @@ describe('getQuantificationSummary', () => {
       grandfatheredTonnesPerYearPerAcreAverage: 0.4199935486561168,
     });
   });
+  it('will get the tonnes that are grandfatherable given a COMET output file for 4 grandfatherable years', async () => {
```

Comment:
I created this test by taking the existing test for processing a soilmetrics output file with an expectation that was created using 5 grandfathered years.  I then examined the actual output when the number of grandfathered years was switched to 4 and found that all of the numbers made sense:
- the switch year moved up by a year
- the outputs for grandfatherable years were only for the 4 relevant years
- the totals and averages differed only by the amount of the now-excluded year (2016)